string NAME=drone_at
bool ERASE_WHEN_FALSE=1

ObjectDroneID drone_id
ObjectLocationID location_id

bool truth_value
